boards/arm/hexiwear_k64/Kconfig.defconfig

```diff
@@ -11,10 +11,10 @@ config BOARD
 	default hexiwear_k64
 
 config HEXIWEAR_DOCKING_STATION
-        bool "Docking Station"
-        default n
-        help
-          Has Hexiwear docking station
+	bool "Docking Station"
+	default n
+	help
+	  Has Hexiwear docking station
```

Comment:
Squash these style fixes into their original commits. In this particular case, the commit that introduced config HEXIWEAR_DOCKING_STATION